requirements_dev.txt

```diff
@@ -15,7 +15,7 @@ django-user-sessions
 
 # Example app (WebAuthn)
 
-webauthn~=1.6.0
+webauthn~=1.11.0
```

Comment:
I find the setup with some dependencies in `setup.py`, then again in `tox.ini` and apparently some more in requirements files quite confusing. I missed this file in #702

Wouldn't it make more sense to make use of the `extras` feature and recommend to install using `pip install -e .[webauthn]` for development?

Reply:
I'll take a look if I have some more time this evening! Probably a good idea to first get a grip on the open PRs, so I'll take a look at those too since my jazzband membership allows you to merge them :smile: